index.html

```diff
@@ -0,0 +1,161 @@
+    <h2 class="header-title-2">TO THE <br> TOURNAMENT <br> OF POWER <br> 2022</h2>
+    <p class="text-1">
+      A gathering of the strongest contestants from 80 nations world-wide,<br> to compete in the first 
+      Tournament Of Power hosted in the universe of DragonBall.
+    </p>
+    <h3 class="header-title-3">2022-11-11(FRI) ~ 13(SUN)</h3>
+    <h4 class="title-location">@CTICC (Cape Town International Convention Centre), GrandWest Arena, Cape Town South Africa</h4>
+  </section>
+
+  <section id="Program" class="container-fluid program-desktop">
+    <h3 class="title-name-2 container-fluid text-white text-center">Main Program</h3>
+    <div class="horizontal-line"></div>
+    <div class="container text-center">
+      <!-- program grid -->
+      <div class="program-grid">
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Warrior Ceremony</h5>
+          <p class="program-description">
+            Introduction of the warriors competing and the hosts for the event.
+          </p>
+        </div>
+        </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Regulations</h5>
+          <p class="program-description">
+            Explaining the scoreboard and rules that will follow for the event.
+          </p>
+        </div>
+      </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Match making</h5>
+          <p class="program-description">
+            Match making tag-team and solo event based on online rating.
+          </p>
+        </div>
+        </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Guest interview</h5>
+          <p class="program-description">
+            Live back and forth open-conversations between guest speakers and audience.
+          </p>
+        </div>
+        </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Raffle trivia</h5>
+          <p class="program-description">
+            Live trivia with a chance to win one of many exculsive prizes.
+          </p>
+        </div>
+        </div>
+      </div>
+      </div>
```

Comment:
- [ ] Please align the items in the cards of the desktop version vertically. You seem to have done it before but broke it along the way
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/190671444-30aa435f-21ba-4744-bfdd-261b71abee8c.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/190671543-47bd5d85-8cc8-4f67-9a65-ec8c8dd73182.png)

Reply:
Solved.